app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -27,7 +27,7 @@
     } %>
   </div>
   <div class="contributions__row" data-dough-component="PopupTip">
-    <p>
+    <p class="callout">
```

Comment:
I don't think this should have a callout class.

Reply:
I discussed the ui with Magda and she thought it was good to emphasise that message a bit.

Reply:
Hmm, I don't really agree with her on that - the callout style has a specific semantic role that this is not required here. Also this hasn't changed as part of this work so I'd prefer to leave it out.